### PR TITLE
animated gif support and debug config improvements

### PR DESCRIPTION
**Pull request by mlamoure**
Added support for animated gif's
Added debug checkbox to plugin config
Added event log messaging upon successful pushover submission
bumped version to 1.5.0

### CODE REVIEW

File: Pushover.indigoPlugin/Contents/Server Plugin/plugin.py [hunk 1]
```diff
@@ -109,6 +114,11 @@ def send(self, pluginAction):
 
 		self.debugLog(u"Result: %s" % r.text)
 
+		if r.status_code == 200:
+			self.logger.info(u"Pushover announceemnt was sent sucessfully, title: " + msgTitle + ", body: " + msgBody)
```
Comment (jcs) [inline Pushover.indigoPlugin/Contents/Server Plugin/plugin.py]: "announcement" is spelled wrong, but should probably be "message" or "notification" anyway

File: Pushover.indigoPlugin/Contents/Server Plugin/plugin.py [hunk 2]
```diff
@@ -85,15 +88,17 @@ def send(self, pluginAction):
 
 		if self.present(pluginAction.props.get('msgAttachment')):
 			attachFile = self.prepareTextValue(pluginAction.props['msgAttachment'])
-			if os.path.isfile(attachFile) and attachFile.lower().endswith(('.jpg', '.jpeg')):
+			extension = os.path.splitext(attachFile)[1]
+
+			if os.path.isfile(attachFile) and (extension == '.jpg' or extension == '.jpeg' or extension == '.gif'):
```
Comment (jcs) [inline Pushover.indigoPlugin/Contents/Server Plugin/plugin.py]: This is no longer lowercasing the filename, so it won't catch images with `.JPG` or similar extensions.

File: Pushover.indigoPlugin/Contents/Server Plugin/plugin.py (hunk 1 above)
Comment (FlyingDiver) [reply Pushover.indigoPlugin/Contents/Server Plugin/plugin.py]: That's already been fixed.  But 'message' would probably be better.

File: Pushover.indigoPlugin/Contents/Server Plugin/plugin.py (hunk 2 above)
Comment (FlyingDiver) [reply Pushover.indigoPlugin/Contents/Server Plugin/plugin.py]: fixed in 1.5.1

File: Pushover.indigoPlugin/Contents/Server Plugin/plugin.py (hunk 1 above)
Comment (FlyingDiver) [reply Pushover.indigoPlugin/Contents/Server Plugin/plugin.py]: fixed (again) in 1.5.1

File: Pushover.indigoPlugin/Contents/Server Plugin/plugin.py (hunk 2 above)
Comment (mlamoure) [reply Pushover.indigoPlugin/Contents/Server Plugin/plugin.py]: Thanks guys!

**Pull request merged**